package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "egg-sequelize",
-  "version": "4.3.1",
```

Comment:
please don't modify version here. others LGTM

Reply:
@akshayKrSingh can you please fix it?